### PR TITLE
[AI Tools] - S3 compliance

### PR DESCRIPTION
**Pull request by eleapttn**
New doc on how to use S3 buckets with AI Tools:

*This doc is required for this [PR 4605](https://github.com/ovh/docs/pull/4605).*

### CODE REVIEW

File: pages/platform/ai/gi_08_s3_compliance/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+
+- A **Public Cloud project**
+- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB)
+- A working `ovhai` CLI. See our guide on [how to install ovhai CLI](https://docs.ovh.com/gb/en/publiccloud/ai/cli/install-client).
+
+## Instructions
+
+In the next steps, you will be able to **create a user to manage your S3 buckets**, **add a new datastore** dedicated to your buckets, then **create and increment a bucket**.
+
+Then, you will be able to use this bucket with the different **OVHcloud AI Tools** through the `ovhai` CLI.
```
Comment (MathieuBsqt) [inline pages/platform/ai/gi_08_s3_compliance/guide.en-gb.md:26]: And also through the OVHcloud control panel ?

File: pages/platform/ai/gi_08_s3_compliance/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+
+Now that you have a dedicated datastore, you can create some S3 buckets through OVHcloud Control Panel and `ovhai` CLI.
+
+### Create an S3 bucket
+
+You have two ways to [create a new S3 bucket](/pages/cloud/storage/object_storage/s3_create_bucket). The first one is to use the UI.
+
+#### Using the Control Panel
+
+To create an **Object Storage bucket**, first log in to your [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager\&from=https://www.ovh.co.uk/\&ovhSubsidiary=GB) and open your `Public Cloud` project. Click on `Object Storage` in the side bar and then on the `My containers` tab. Now, you can click on `Create Object Container`.
```
Comment (MathieuBsqt) [inline pages/platform/ai/gi_08_s3_compliance/guide.en-gb.md]:
```suggestion
To create an **Object Storage bucket**, first log in to your [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager\&from=https://www.ovh.co.uk/\&ovhSubsidiary=GB) and open your `Public Cloud` project. Click on `Object Storage` in the side bar and then on the `My containers` tab. Now, click the `Create an object container` button.
```

**Review by MathieuBsqt — COMMENTED** [review]
Great work 👏! Here are some suggestions, feel free to apply changes or not ;) !

Also data store is supposed to be written in two words to be grammatically correct I think. But in the commands used, it always appears as a single word. (`ovhai datastore list` for example. So I didn't correct it, but we'll need to fix the way we spell it in the text (not in the commands).

**Comment by eleapttn** [comment]
Thanks @MathieuBsqt.

FYI: `ovhai data store list` is depreciated. Here is the right command: `ovhai datastore list`

🚀

**Comment by OVH-Team-Guides** [comment]
Hello @eleapttn , could you please provide us with the following in FR? Thanks in advance

- title
- excerpt

**Comment by eleapttn** [comment]
Thanks for the review @OVH-Team-Guides!

_French translations here:_
titre: Données - Conformité S3 avec les AI Tools
extrait: Apprendre à utiliser les buckets S3 avec les AI Tools

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @eleapttn

**Pull request merged**